client/landing/gutenboarding/utils.ts

```diff
@@ -13,7 +6,6 @@ import { SiteVertical } from './stores/onboard/types';
 /**
  * ⚠️😱 Calypso dependencies 😱⚠️
  */
-import wpcom from '../../lib/wp';
 import { untrailingslashit } from '../../lib/route';
 import { urlToSlug } from '../../lib/url';
```

Comment:
Hmmmn I think it would be easier if the backend returned the site_slug since it's so useful. 

I'll create a patch.

Reply:
D38739-code

Reply:
Thanks for helping with this! We'll probably end up using the slug to show to the user the domain he chose previously so it's good to have it. 
For now I achieved the needed functionality with `blogId`: https://github.com/Automattic/wp-calypso/pull/39391/commits/4c30c3fd2524a52448d9b5c1cf009636c81fd273